EXILED/Exiled.Events/EventArgs/Player/DroppingAmmoEventArgs.cs

```diff
@@ -36,14 +39,21 @@ public class DroppingAmmoEventArgs : IPlayerEvent, IDeniableEvent
         /// <param name="isAllowed">
         /// <inheritdoc cref="IsAllowed" />
         /// </param>
-        public DroppingAmmoEventArgs(Player player, AmmoType ammoType, ushort amount, bool isAllowed = true)
+        public DroppingAmmoEventArgs(Player player, byte itemType, AmmoType ammoType, ushort amount, bool isAllowed = true)
```

Comment:
```suggestion
        public DroppingAmmoEventArgs(Player player, ItemTypeitemType, ushort amount, bool isAllowed = true)
```
(require to sligly change Transpiller to remove GetAmmoType)